Makefile

```diff
@@ -128,7 +128,7 @@ $(DOCS_SYNOPISIS_DIR)/*.md: vendor $(ADDON_ASSET_FILE)
 synopsis_docs: $(DOCS_SYNOPISIS_DIR)/*.md
 
 .PHONY: prerelease
-prerelease: clean fmtcheck test cross
```

Comment:
We should keep cross here as well. As you say, we want to make sure that cross compilation works beforehand